powersimdata/design/transmission/upgrade.py

```diff
@@ -293,6 +293,17 @@ def _identify_mesh_branch_upgrades(
         raise ValueError(err_msg)
 
     # Calculate selected metric for congested branches
+    if method == "cost":
+        # Calculate costs for an upgrade dataframe containing only composite_allow_list
+        base_grid = Grid(
+            ref_scenario.info["interconnect"], ref_scenario.info["grid_model"]
+        )
+        base_grid.branch = base_grid.branch.filter(items=congested_indices, axis=0)
+        upgrade_costs = _calculate_ac_inv_costs(base_grid, sum_results=False)
+        # Merge the individual line/transformer data into a single Series
+        merged_upgrade_costs = pd.concat(
+            [v.Cost for v in upgrade_costs.values()], sort=True
+        )
```

Comment:
What we are trying to achieve by passing `sort=True` here?

Reply:
Originally in the `branch` data frame, branch IDs are sorted. If we do not add `sort=True` here, then we will get either all Line branch IDs and then all Transformer/TransformerWinding branch IDs in the concatenated data frame, or vice versa. Now that I think about it though, it probably doesn't matter, because pandas should automatically use the indices for the element-wise division, and we never use `merged_upgrade_costs` after that. I will verify and remove.

Reply:
Agree with your second point, I think it won't matter given the pandas automatic index matching. The main concern I had here is `sort=True` in pd.concat will only sort non-concatenation axis (not the values), here is the column, which is trivial given there is only one column in pd.Series.

Reply:
Confirmed: quick testing showed no change in with vs. without the `sort=True`, so it's been removed.